webhook/env.go

```diff
@@ -82,3 +84,15 @@ func TLSMinVersionFromEnv(defaultTLSMinVersion uint16) uint16 {
 		panic(fmt.Sprintf("the environment variable %q has to be either '1.2' or '1.3'", tlsMinVersionEnvKey))
 	}
 }
+
+func DisableNamespaceOwnershipFromEnv() *bool {
+	disableNamespaceOwnership := os.Getenv(disableNamespaceOwnershipEnvKey)
+	if disableNamespaceOwnership == "" {
+		return nil
```

Comment:
Can we default this as `false` and have the return type be `bool`

Reply:
@dprotaso the behavior I was intending was that `WEBHOOK_DISABLE_NAMESPACE_OWNERSHIP` overrides the `DisableNamespaceOwnership` webhook option __only__  when the env-var is set.

To do this, I need to be able to distinguish between an unset env-var and a `false` one:

https://github.com/knative/pkg/pull/3103/files#diff-71e3f02004a234858e10c42bf73e303ba86e5000ad4a1a95ccf5f6d94447c5e8R169-R174